lib/view_backed/version.rb

```diff
@@ -1,3 +1,3 @@
 module ViewBacked
-  VERSION = "0.0.2"
+  VERSION = "0.0.3"
```

Comment:
I ended up bumping the version number in this PR because changes to this repo are fairly infrequent. I imagine if there were more frequent updates, then we'd want to have a separate commit bumping the version number to lump changes together.